lib/tasks/add_german_topics.rake

```diff
@@ -0,0 +1,43 @@
+namespace :sensorstory do
+
+  desc 'Add or rename a list of german topics'
+  task add_german_topics: :environment do
+    Topic.transaction do
+      not_matched = [
```

Comment:
3 things: 1. Structure content (eg for search), 2. Control Chatbot, 3. Define Icons for Timestamps